sqe-http-api/DTOs/ImagedObject.cs

```diff
@@ -20,43 +20,5 @@ public class ImagedObjectDTO
     public class ImagedObjectListDTO
     {
         public List<ImagedObjectDTO> imagedObjects { get; set; }
-    };
-
-    public static class ImagedObjectIdFormat
```

Comment:
As agreed, we no longer need this function, and it should not have been in the DTO anyway.  It is removed now.